### ISSUE TITLE
Build error on android after upgrade to gradle 7

### ISSUE THREAD

**Issue by gax97** [auth0/react-native-auth0#446]
### Describe the problem

After upgrading my android project to use gradle 7 I have a problem building it because of the error in react-native-auth0
```
error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDevDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/danilogacevic/Documents/app/node_modules/react-native-auth0/android/build.gradle' line: 94

* What went wrong:
A problem occurred configuring project ':react-native-auth0'.
> Configuration with name 'compile' not found.

```

I tried commenting out this line: 94 and when I do so build succeeded.

### What was the expected behavior?

I expect to be able to build android project

### Reproduction
In `build.gradle`
Update `buildToolsVersion` and `compileSdkVersion` to 31
Update gradle to `classpath 'com.android.tools.build:gradle:7.0.3'`

### Environment
React-Native/Android

**Comment by poovamraj** [comment]
Hi @gax97 Can you confirm the React Native version you are using?

**Comment by gax97** [comment]
Hi @poovamraj I am using react-native 0.66.3 and react-native-auth0 2.11.0

**Comment by poovamraj** [comment]
This issue is fixed in #450 and will be released in a new version. We will keep you updated on the release

**Comment by param-finder** [comment]
I'm facing the exact same issue. I'm using Gradle 7.0.2 and react-native-auth0 2.11.0

```
FAILURE: Build failed with an exception.

* Where:
Build file '~/Documents/Test/node_modules/react-native-auth0/android/build.gradle' line: 94

* What went wrong:
A problem occurred configuring project ':react-native-auth0'.
> Configuration with name 'compile' not found.
```

**Comment by poovamraj** [comment]
Hey all 👋  The fix for this should be available in v2.13.0 which is currently live. We are closing this issue for now but please feel free to open it back if you face further issues
